tests/integration/Paginator/Adapter/QueryBuilder/GetSetLimitCest.php

```diff
@@ -68,5 +69,21 @@ public function paginatorAdapterQuerybuilderGetSetLimit(IntegrationTester $I)
         );
 
         $I->assertEquals($paginator, $setterResult);
+
+
+
+        try {
+            $paginator = new QueryBuilder(
+                [
+                    'builder' => $builder,
+                    'limit'   => 0,
+                    'page'    => 1,
+                ]
+            );
+
+            $paginator->paginate();
+        } catch (Exception $ex) {
+            $I->assertEquals('Limit must be greater then zero', $ex->getMessage());
+        }
```

Comment:
If you want to check message or exception code, you can pass them with exception instance:
```php
// will check that exception MyException is thrown with "Don't do bad things" message
$I->expectException(new MyException("Don't do bad things"), function() {
    $this->doSomethingBad();
});
```

Reply:
There is also `expectThrowable`. It handles and checks throwables (Exceptions/Errors) called inside the callback function. Either throwable class name or throwable instance should be provided:

```php
$I->expectThrowable(MyThrowable::class, function() {
    $this->doSomethingBad();
});

$I->expectThrowable(new MyException(), function() {
    $this->doSomethingBad();
});
```

Reply:
Do not use try/catch in tests. If `try` will not raise exception test always will pass, isn't? ;)

Reply:
Phpstorm noticed that 'expectException' is deprecated so I tried a different method. Didn't know you could use 'expectThrowable'.